modules/nodes-pods-configmap-creating-from-directories.adoc

```diff
@@ -1,30 +1,48 @@
 // Module included in the following assemblies:
 //
-//* authentication/configmaps.adoc
+//* nodes/pods/nodes-pods-configmap.adoc
 
 :_content-type: PROCEDURE
 [id="nodes-pods-configmap-creating-from-directories_{context}"]
 = Creating a config map from a directory
 
-You can create a config map from a directory. This method allows you to use multiple files within a directory to create a config map.
+You can create a config map from a directory by using the `--from-file` flag. This method allows you to use multiple files within a directory to create a config map.
 
-.Procedure
+Each file in the directory is used to populate a key in the config map, where the name of the key is the file name, and the value of the key is the content of the file.
 
-The following example procedure outlines how to create a config map from a directory.
+For example, the following command creates a config map with the contents of the `example-files` directory:
 
-. Start with a directory with some files that already contain the data with which you want to populate a config map:
-+
 [source,terminal]
 ----
-$ ls example-files
+$ oc create configmap game-config --from-file=example-files/
 ----
-+
+
+[source,terminal]
+----
+$ oc describe configmaps game-config
+----
+
 .Example output
 [source,terminal]
 ----
-game.properties
-ui.properties
+Name:           game-config
+Namespace:      default
+Labels:         <none>
+Annotations:    <none>
+
+Data
+
+game.properties:        158 bytes
+ui.properties:          83 bytes
 ----
+
+You can see that the two keys in the map are created from the file names in the directory specified in the command. Because the content of those keys might be large, the output of `oc describe` only shows the names of the keys and their sizes.
```

Comment:
Is it always the case that `oc describe` only shows the names and their sizes or is that just the case of this example? I was thinking if it is always the case, it might be more clear to say:
```suggestion
You can see that the two keys in the map are created from the file names in the directory specified in the command. The content of those keys might be large, so the output of `oc describe` only shows the names of the keys and their sizes.
```